packages/react-percy-webpack/src/resolve/getExtension.js

```diff
@@ -10,5 +9,5 @@ export default function getExtension(configPath) {
       return extension;
     }
   }
-  return path.extname(configPath);
+  return '.babel.js';
```

Comment:
Maybe add your PR comment above about why we're doing this as a code comment here?

Reply:
Good idea, will do before I merge this